server/routes/termdb.cluster.ts

```diff
@@ -287,6 +287,30 @@ async function validateMetaboliteIntensityNative(q: MetaboliteIntensityQueryNati
 		}
 	}
 
+	q.find = async (param: TermdbClusterRequest) => {
```

Comment:
it doesn't make sense to use TermdbClusterRequest type of parameter. try to move this initiator function out of `routes/termdb.cluster.ts` and into `mds3.init.js`

Reply:
the hierachical clustering will be implemented to metabolite data. Do you think  we should not use TermdbClusterRequest as the parameter for metabolites data?

Reply:
I think he means that the find method is not going to be called from the client so the input parameter does not have to be a request, the get does have to be a request as it is called from the client to do the hierarchical clustering.

Reply:
we copy pasted from the get method as the input is very similar, that is why we used the request param, we can improve it.